### DISCUSSION TITLE
Disabling ports when connected already.

### DISCUSSION THREAD

**Discussion by gintarasgraciovas** [clientIO/joint#2809] [Q&A] [answered]
### Introduction

I'm currently trying to disable ability to create a link from already connected ports. I assume that there is such functionality, but I can find it. Please help :)

### Steps to reproduce

1) add 2 elements to the graph
2) connect them
3) try to create a link from the port which is connected already

Observe that link is being created when it shouldn't.

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS, JointJS+

**Comment by kumilingus** [comment, marked as answer]
For that, you can use `validateMagnet()` paper option. Here's an example: https://www.jointjs.com/demos/working-with-ports
